### PR TITLE
Adds a useful script for cleaning up unused test orgs

### PR DESCRIPTION
**Pull request by AP-Hunt**
What
----

Our continuous smoke tests, acceptance tests and the like
create organizations in CF, and provision backing services.
Sometimes the tests fail and the resources don't get cleaned
up. This script tidies up.

How to review
-------------
1. Code review
2. Try it out in `dev02`

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨

### CODE REVIEW

File: scripts/cleanup_leftover_test_orgs.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
+
+if [[ "$(cf api | head -n1 | awk '{print $3}')" =~ ^.*cloud.service.gov.uk$ ]]; then
+  echo "Don't try to use this script in prod. It's a bit too risky."
```
Comment (LeePorte) [inline scripts/cleanup_leftover_test_orgs.sh:6]: Actually given that there is later confirmation, I disagree

Comment (AP-Hunt) [reply scripts/cleanup_leftover_test_orgs.sh:6]: Fair point!

**Review by paroxp — APPROVED** [review]
Few considerations.

 If it has to be used by operators, why wouldn't we want that functionality in pazmin, so Mark could also have a go?
 
 And if it's intends to be automated, should we also be adding manually triggerable pipeline?
 
 In general, good and much needed idea.

**Review by schmie — APPROVED** [review]
:+1: works like a charm (tested pagination in my dev by reducing to 10).

**Pull request merged**